burst_detector/__init__.py

```diff
@@ -14,12 +14,10 @@
     cross_proj,
```

Comment:
this import can be removed with the function since `cross_sim` isn't in `stages.py` anymore